languages/en.ini

```diff
@@ -1237,10 +1237,10 @@ Username cannot be blank = "Username cannot be blank"
 Username is already in use in another library card = "Username is already in use in another library card"
 verification_done = "Your email address has been verified successfully."
 verification_email_change_sent = "Email address verification instructions have been sent to the new email address. You must verify the address before the change will take effect."
-verification_email_notification = "A request was just made to verify your email address for your account with %%library%%."
+verification_email_notification = "A request was just made to verify your email address for your account with %%library%%. Please ignore this message if you have not added or changed your email address."
```

Comment:
It might be better to set separate translation strings for the newly added sentences and add them separately in the email template. That will make it easier to ensure that all of the translations remain consistent with each other.

Reply:
I decided to roll back most of the changes. I couldn't come up with meaningful translation keys, and we can always tweak the texts locally.

Reply:
Thanks, @EreMaijala. I think those changes are useful, but it is probably less work to keep them as local string customizations for the moment. Happy to revisit in the future (and/or brainstorm key names) if that would be helpful, though!